### ISSUE TITLE
Add query duckdb version function

### ISSUE THREAD

**Issue by Ethanzjp** [duckdb/duckdb#1521]
# Feature Request

## Summary 
Is your feature request related to a problem? Please describe:

In sqlite3,
```shell
sqlite> select sqlite_version();
3.31.1
```

I test the duckdb,

```C++
#include <duckdb.hpp>
int
main(int argc, char** argv){
    duckdb::DuckDB db(nullptr);
    duckdb::Connection con(db);
    auto result = con.Query("SELECT duck_version();");
    result->Print();
	return 0;
}
```
**or**

```C++
#include <duckdb.hpp>
int
main(int argc, char** argv){
    duckdb::DuckDB db(nullptr);
    duckdb::Connection con(db);
    auto result = con.Query("SELECT duckdb_version();");
    result->Print();
	return 0;
}
```

**or**

```C++
#include <duckdb.hpp>
int
main(int argc, char** argv){
    duckdb::DuckDB db(nullptr);
    duckdb::Connection con(db);
    auto result = con.Query("SELECT  version();");
    result->Print();
	return 0;
}
```

```shell
Catalog Error: Scalar Function with name duckdb_version does not exist!
Did you mean "decode"?
LINE 1: SELECT duckdb_version();
```

Is it possible to add support for this `version` function in `DuckDB`?

**Comment by Alex-Monahan** [comment]
Hello!

There is a way to pull the DuckDB version that you can use at the moment if
that works! Does this help?

https://github.com/cwida/duckdb/pull/808

select * from pragma_version();

**Comment by EthanDBer** [comment]
> select * from pragma_version();

Maybe a simple function to get the version should be added. Of course it works for `select * from pragma_version();`

**Comment by hannesmuehleisen** [comment]
What do you mean by "simple"? Does not get much more simple than `pragma_version()`. Do you mean a scalar function?

**Comment by EthanDBer** [comment]
> What do you mean by "simple"? Does not get much more simple than `pragma_version()`. Do you mean a scalar function?

yes, I means scalar function.

**Comment by hannesmuehleisen** [comment]
You can also use a subquery to make it a scalar: `SELECT (SELECT library_version FROM pragma_version()) v;`

**Comment by Mytherin** [comment]
Perhaps we could add a macro that maps sqlite_version to that subquery, for further SQLite compatibility?

**Comment by EthanDBer** [comment]
> Perhaps we could add a macro that maps sqlite_version to that subquery, for further SQLite compatibility?

I think this is a good idea.

**Comment by EthanDBer** [comment]
Is this suitable for duckdb's begginer? I want to try if it is possible. Of course I hope you can provide some help. :)

**Comment by hannesmuehleisen** [comment]
Yes, that is certainly a good issue to begin with. Happy to review a PR!

**Comment by Mytherin** [comment]
> 
> 
> Is this suitable for duckdb's begginer? I want to try if it is possible. Of course I hope you can provide some help. :)

I would have a look at the default functions [here](https://github.com/cwida/duckdb/blob/master/src/catalog/default/default_functions.cpp). It should not be so difficult to add a macro for this :)

**Comment by zippond** [comment]
As a beginner I want to try this one, but sadly find it's has been fixed in #1847. Maybe this one should be closed now.

**Comment by Ethanzjp** [comment]
> As a beginner I want to try this one, but sadly find it's has been fixed in #1847. SELECT version(); Maybe this one should be closed now.

Thanks!
